data/capture_config.yaml

```diff
@@ -0,0 +1,26 @@
+spoof_regex: .*
+ntlm_challenge: "1122334455667788"
+ntlm_domain: anonymous
+services:
+  DRDA: off
```

Comment:
I think @jmartin-r7 forgot to press submits on his review comments that we went through together, I'll try to throw down duplicate comments for the one I remember :+1:

It'd be great to provide a comment/context as to why these are off by default 👀

Reply:
Yeah, honestly, no reason other than my bias of not having needed them before; but might as well have them on. I'll put them on by default.